README.md

```diff
@@ -42,6 +42,11 @@ Add jfrog-client-js as a dependency to your package.json file:
 - [Platform](#platform)
   - [Register For Web Login](#register-for-web-login)
   - [Get Access Token From Web Login](#get-access-token-from-web-login)
+- [Xsc](#xsc)
```

Comment:
I'd add Xray-Source-Control, it's not a well-known term. yet:)

Reply:
changed